rfcs/text/0001-rfc-process.md

```diff
@@ -70,103 +84,111 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
+   1. Fork the HumanCellAtlas `dcp-community` repository
+   2. Copy `rfcs/rfc-template.md` to `rfcs/text/0000-my-feature.md`. Make `my-feature` descriptive but do not assign an RFC number yet.
+   3. Fill out the Problem Statement and Author(s) sections.
+   4. Push the changes to your branch and create a _Draft_ PR.
+   5. Label the PR with the "rfc-draft" label.
+
+#### **Non-Authors**:
+
+When a _Draft_ RFC PR is published and labeled with "rfc-draft," specific commentary is NOT RECOMMENDED (negative or positive) until the Authors(s) propose the RFC for community review. If the draft RFC sparks interest, it is RECOMMENDED that one reach out to the Author(s) offline to suggest a collaboration.
 
-      For governance RFCs:
-        
-      - Add the "rfc-community-review" and the "Governance" labels. 
+### Proposing RFCs
+
+#### **Authors**:
 
-- Add a minimum two week _last call_ for the completion of the Community review to the top-level summary comment in the RFC pull request.
+  - Fill in the RFC with attention to detail
+  
+  - Submit a pull request. If the PR was originally in a draft state, remove the "rfc-draft" label.
+  
+  - Add the "rfc-community-review" label.
+  
+  - Find the auto-generated ticket in the Zenhub `dcp-community` board and assign all Author(s) to the ticket.
+  
+  - Add a minimum two week _last call_ for the completion of the Community review to the top-level Summary section in the RFC pull request.
 
     **EXAMPLE** 
 
     **April 1**: Last call for community review
 
-- Request a Community review of the RFC on the HumanCellAtlas **#dcp** slack channel. Include a link to the RFC pull request and the last call deadline:
+  - Request a Community review of the RFC on the HumanCellAtlas **#dcp** Slack channel. Include a link to the RFC pull request and the last call deadline:
 
     **EXAMPLE** 
   
-     ***@channel**: Call for community review of the RFC process - https://github.com/HumanCellAtlas/dcp-community/pull/27 - Last call is April 1.*
+    ***@channel**: Call for community review of the RFC process - https://github.com/HumanCellAtlas/dcp-community/pull/27 - Last call is April 1.*
+  
+  - For **Software RFCs**:
+    
+    - Add the appropriate software component name label to the RFC OR if the RFC impacts multiple DCP software components, add the "Architecture" label instead. If uncertain about the appropriate project name, then ask on the **#dcp-project-mgmt** Slack channel.
+    
+    - Link the ticket that was auto-generated in the `dcp-community` Zenhub board to an open ticket in the `dcp` repo. RFCs generally fulfill a deliverable for a Spike ticket and linking the open ticket in the `dcp` repo to the auto-generated PR ticket guarantees a paper-trail of the work.
```

Comment:
This section is somewhat redundant with the _spike_ details at Line 91?

You could also reference [Modeling Objectives Requiring Further Research](https://github.com/HumanCellAtlas/dcp-community/blob/master/rfcs/text/0012-roadmaps%2Bplanning.md#modeling-objectives-requiring-further-research). 

I think that we need a bit more clarity about how the bi-directional links between RFCs and their Spikes should be modeled.

Reply:
Removed the redundant sentence.

At this moment, I've chosen not to reference Modeling Objectives. I think I'd like to keep the task list as clean as possible.